config/migrations/2023/lpdc/20230512120000-add-concept-display-configurations-for-instanced-concepts.sparql

```diff
@@ -0,0 +1,33 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX cpsv: <http://purl.org/vocab/cpsv#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX lpdcExt: <https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX pav: <http://purl.org/pav/>
+
+INSERT {
+  GRAPH ?graph {
+    ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+    ?configUri a lpdcExt:ConceptDisplayConfiguration ;
+      mu:uuid ?configId ;
+      lpdcExt:conceptIsNew false ;
+      lpdcExt:conceptInstantiated true ;
+      dct:relation ?eenheid .
+  }
+}
+WHERE {
+  GRAPH ?graph {
+    ?instance a cpsv:PublicService;
+      dct:source ?concept;
+      pav:createdBy ?eenheid.
+  }
```

Comment:
If a bestuurseenheid has multiple instances, the same concept will be returned here, which means multiple Configs would be inserted in the DB. The current configId kind of bypasses that since the uri would be the same for each concept duplicate.

I tried using a subquery to only select the unique concepts and the end result seemed the same, not sure if that's the correct solution here.

Reply:
I solved this in a different way that sidestepped this issue.